package.json

```diff
@@ -123,8 +158,10 @@
     "eslint-plugin-vue": "^6.0.1",
     "eslint-plugin-wdio": "^5.13.2",
     "eslint-plugin-xss": "^0.1.9",
+    "jstransformer-babel": "^2.1.0",
     "node-fetch": "^2.6.0",
     "npm": "^6.13.4",
+    "sort-package-json": "^1.31.0",
```

Comment:
😂

Reply:
It will run automatically as pre-commit hook 😎